bin/hdfcoinc/pycbc_multiifo_add_statmap

```diff
@@ -11,6 +11,17 @@ import pycbc.conversions as conv
 from pycbc.events import coinc
 from ligo import segments
 
+def get_ifo_attr(fi):
+    if 'ifos' in fi.attrs:
```

Comment:
maybe this should be a try-except, if eventually we want to be in a position where every input file has the expected attrs entry and only obsolete ones need the workaround